speakrune/gen/speakrune.hoon

```diff
@@ -0,0 +1,51 @@
+|=  raw=tape
+^-  tape
+=<  ?:((valid raw) (turn raw convert) "input contains non-rune characters")
+|%
+++  valid
+  |=  x=tape
+  ^-  ?
+  =/  chars  (sy x)
+  (~(all in chars) |=(c=@t (~(has by table) c)))
+++  convert
+  |=  x=@t
+  ^-  @t
+  =/  r  (trip (~(got by table) x))
+  (crip r)
```

Comment:
Looks like you could just produce `(~(got by table) x)`, instead of converting it to tape and then back to `@t`?

Then, as with the `~(all in chars)` case above, you could just say `++  convert  ~(got by table)`.

Reply:
Yeah, this was a leftover from me adding an ace separator between the output triplets. After looking at the docs I realized they didn't use a separator but then I didn't fully clean up the code.